client/src/components/MapElement/MapElement.js

```diff
@@ -11,32 +13,88 @@ MapElement.propTypes = {
 export function MapElement({ className }) {
   const appView = useSelector((state) => state.view.appView)
   const activeTripId = useSelector((state) => state.view.activeTripId)
+  const dispatch = useDispatch()
+
+  const [destination, setDestination] = useState('')
+  const [stagesPerDay, setStagesPerDay] = useState('')
+  const [budget, setBudget] = useState('')
+  const [numberOfDays, setNumberOfDays] = useState('')
+  const [invalidForm, setInvalidForm] = useState(false)
+
+  const resetForm = () => {
+    setDestination('')
+    setStagesPerDay('')
+    setBudget('')
+    setNumberOfDays('')
+    setInvalidForm(false)
+  }
+  const handleCloseNewTripModal = () => {
+    dispatch(setAppView(AppView.TRIP_VIEW))
```

Comment:
Ah I get what you mean now. 

Maybe if they exit by clicking the bg, we should stay in the globe mode but just close the modal? And only go to TRIP_VIEW if they press "brew/submit"?

Reply:
i implemented this, not sure if i broke sth tho